### ISSUE TITLE
ScrollView widget

### ISSUE THREAD

**Issue by rdbende** [tukaan/tukaan#76]
A scrollable frame with.

```python
scrollview = ScrollView(
    app,
    overflow=(False, "auto"),
    padding=(1, 2, 3, 4),
)
```

**Comment by Moosems** [comment]
How does it now when to stop the scrolling?

**Comment by sumeshir26** [comment]
> How does it now when to stop the scrolling?

No, the user will scroll the frame

**Comment by Moosems** [comment]
Here is what I meant to ask: Is there a limit to how long/much they can scroll? Will it limit it to the length/width of the widgets inside?

**Comment by rdbende** [comment]
> Is there a limit to how long/much they can scroll? Will it limit it to the length/width of the widgets inside?

No, it won't.

**Comment by Moosems** [comment]
Ok 👍  Is there a way to add a limit?

**Comment by rdbende** [comment]
Yes, I could implement some limit control, but I don't really see its benefit 🤔

**Comment by Moosems** [comment]
So that the user could make something similar to a webpage but not let the user scroll out into infinity

**Comment by Moosems** [comment]
> Na yeah, neither thought I to scroll into infinity. Maybe I was a bit ambiguous. So it can only scroll until there's content, and not more (so that's why I didn't understand you limit question, there's no need for it).

That makes more sense. Thanks for the clarification!

**Comment by Moosems** [comment]
I have a basic Scrollable Frame in one of my own projects that could be forged but I still need help fixing some kinks with it.

**Comment by sumeshir26** [comment]
Wouldn't that basically be a Frame with a scrollbar binded to it?

**Comment by Moosems** [comment]
No, they don't scroll so you have to make a solution with a canvas and inner frame. It's weird.
